HW14/part2/Box.java

```diff
@@ -0,0 +1,127 @@
+package part2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Box<T> {
```

Comment:
'<T>` means that `Box could be any type. The correct way is `Box<T extends Fruit>`.